### DISCUSSION TITLE
Welcome to lwjson Discussions!

### DISCUSSION THREAD

**Discussion by MaJerle** [MaJerle/lwjson#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lvasiljevic** [comment]
Add in README how to install fast your library. or some make ; make install

**Comment by MaJerle** [comment]
https://docs.majerle.eu/projects/lwjson/en/latest/get-started/index.html#getting-started
